menu.h

```diff
@@ -27,6 +27,9 @@ class menu
   /// Get one button at index i
   menu_button &get_button(int index);
 
+  /// Get one button at index i
+  menu_button &get_button(const std::string& label);

```

Comment:
As stated above: add the `const` member function of the same name here :-)

Reply:
Couldn't get it to work :/